docs/src/main/asciidoc/security-openid-connect-client-quickstart.adoc

```diff
@@ -0,0 +1,506 @@
+////
```

Comment:
Maybe I wasn't clear but this one shouldn't have the `-quickstart` suffix. And the links to it should be updated.

Reply:
@gsmet I see, got it, renamed it